WordPress/src/main/java/org/wordpress/android/ui/main/jetpack/staticposter/JetpackStaticPosterFragment.kt

```diff
@@ -47,6 +51,21 @@ class JetpackStaticPosterFragment : Fragment() {
         }
     }
 
+    @Composable
+    fun LockScreenOrientation(orientation: Int) {
```

Comment:
Although this has the potential to be reused I would mark it as `private` for now as it seems the comment inside `onDispose` makes it clear this should only be used in this specific scenario because there are safeguards in place to restore the orientation later.

Reply:
I didn't catch this, nor did detkt. :( If CI wasn't so dang slow, I would change it. I can do a PR to trunk to capture this change this and the informative note above. 👍